### DISCUSSION TITLE
Might __set_callbacks function in RemoteSlaveContext class  have some bug?

### DISCUSSION THREAD

**Discussion by Ch0pperX** [pymodbus-dev/pymodbus#1667] [General]
When I followed forwarder.py example, the `write_coil` would transfer to `write_coils`. 
I found "c5" which modbus function code = 0x5 would call `write_coils` to forward the message to the client instead of `write_coil`.

Would there have any effect if I revise to `write_coil`?

**Comment by janiversen** [comment]
I dont think so, looks like a bug. Pull requests are welcome.

**Comment by janiversen** [comment]
This is solved.
